lib/metro_2/records/k2_segment.rb

```diff
@@ -0,0 +1,20 @@
+module Metro2::Records
+  class K2Segment < Record
+    LENGTH = 34
+    POSSIBLE_INDICATORS = [1, 2, 9].freeze
```

Comment:
I think it might helpful to add a comment here.
"1 = Purchased From Name
2 = Sold To Name
9 = Remove Previously Reported K2 Segment Information"

Reply:
I moved these values into a constant to mirror other uses.